src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -520,7 +520,7 @@ internal class PermissionTest : DatabaseTest() {
   }
 
   @Test
-  fun `contributors have access to data associated with their organizations`() {
+  fun `contributors have access to data associated with their organizations, but cannot update or delete accessions`() {
```

Comment:
This feels too specific to me -- when we make a similar change for nursery support, we'd kind of need to update this to also describe the nursery-related limitations.

How about something like, `contributors have full read and limited write access to data associated with their organizations`?

Reply:
Yeah, that sounds like good verbiage. I'll go with it!